test/liveValidatorTests.ts

```diff
@@ -226,6 +232,42 @@ describe("Live Validator", () => {
       await validator.initialize()
       assert.notStrictEqual(validator.cache["microsoft.batch"], undefined)
     })
+    it("should initialize and ignore certain swaggers by default", async () => {
+      const options = {
+        directory: "./test/liveValidation/swaggers/specification",
+        swaggerPathsPattern: ["batch/**/*.json"]
+      }
+      const validator = new LiveValidator(options)
+      await validator.initialize()
+      assert.strictEqual(1, Object.keys(validator.cache).length)
+      const microsoftBatch = validator.cache["microsoft.batch"]
+      if (microsoftBatch === undefined) {
+        throw new Error("microsoftBatch === undefined")
+      }
+      assert.strictEqual(undefined, microsoftBatch["2019-08-01.10.0"])
+      assert.strictEqual(7, microsoftBatch["2017-01-01"].get.length)
+      assert.strictEqual(2, microsoftBatch["2017-01-01"].patch.length)
+      assert.strictEqual(4, microsoftBatch["2017-01-01"].post.length)
+    })
+    it("should not ignore any swagger paths if options delcare no ignore path", async () => {
+      const options = {
+        directory: "./test/liveValidation/swaggers/specification",
+        excludedSwaggerPathsPattern: [],
+        swaggerPathsPattern: ["batch/**/*.json"]
+      }
+      const validator = new LiveValidator(options)
+      await validator.initialize()
+      assert.strictEqual(1, Object.keys(validator.cache).length)
+      const microsoftBatch = validator.cache["microsoft.batch"]
+      if (microsoftBatch === undefined) {
+        throw new Error("microsoftBatch === undefined")
+      }
+      // Should find two different api versions, one for data plane and one for management plane
+      assert.strictEqual(1, microsoftBatch["2019-08-01.10.0"].get.length)
+      assert.strictEqual(7, microsoftBatch["2017-01-01"].get.length)
+      assert.strictEqual(2, microsoftBatch["2017-01-01"].patch.length)
+      assert.strictEqual(4, microsoftBatch["2017-01-01"].post.length)
+    })
     it("should initialize for all swaggers", async () => {
```

Comment:
As synced offline, pls add a test to verify the assigned excludedSwaggerPathsPattern value.